src/pyprojectx/initializer/initializers.py

```diff
@@ -157,7 +157,10 @@ def initialize_global(options):
         print(
             f"Add the scripts to your PATH, f.e. by appending following to your shell's profile"
             f" ({RESET}~/.profile{BLUE}, {RESET}~/.zshrc{BLUE}, {RESET}~/..bashrc{BLUE}, ...):\n"
-            f"{RESET}export PATH=$PATH:{global_dir.parent.absolute()}, file=sys.stderr"
+            f"{RESET}export PATH=$PATH:{global_dir.parent.absolute()}"
+            f"\nOr for xonsh's ~/.xonshrc append the following\n"
+            f"{RESET}$PATH.append('{global_dir.parent.absolute()}')",
+            file=sys.stderr
```

Comment:
I also noticed that the output of stuff had this `file=sys.stderr` which looks like it was meant to be outside of the f string so shifted it there while adding my text but figured I'd point that out in case it was intentional

Reply:
The CI/CD pipeline requires everything to be formatted with black, which would add a comma behind stderr.
Can you run `px black` and commit again?